### PR TITLE
Add devel package for linters

### PR DESCRIPTION
**Pull request by perlpunk**
Related ticket: https://progress.opensuse.org/issues/153793

### CODE REVIEW

File: dependencies.yaml [hunk 1]
```diff
@@ -8,11 +8,12 @@
...
 ---
 targets:
-  cpanfile: [main, test]
-  spec:     [main, test]
+  cpanfile: [main, test, devel]
+  spec:     [main, test, devel]
   cpanfile-targets:
     # target: cpanfile target type (default main)
     test: test
+    devel: develop
```
Comment (b10n1k) [inline dependencies.yaml:16]: Suggestion: I would name it `devel`

**Review by baierjan — APPROVED** [review]

File: dist/rpm/os-autoinst-scripts-deps.spec [hunk 2]
```diff
@@ -36,6 +38,11 @@ Suggests:       postgresql
 - auto-review - Automatically detect known issues in openQA jobs, label openQA jobs with ticket references and optionally retrigger
 - openqa-investigate - Automatic investigation jobs with failure analysis in openQA
 
+%package devel
+Summary:        Development package for os-autoinst-scripts-deps
+Group:          Development/Tools/Other
+Requires:       %devel_requires
+
```
Comment (perlpunk) [inline dist/rpm/os-autoinst-scripts-deps.spec:45]: I'm not sure if this enough to create a new package in OBS

Comment (baierjan) [reply dist/rpm/os-autoinst-scripts-deps.spec:45]: Good point. I believe so. This should create a new subpackage, like in openQA, but you are likely missing a [`%description`](https://github.com/os-autoinst/openQA/blob/6ca685738c5db7ccdd9394dec8d4841a293c65ec/dist/rpm/openQA.spec#L170) and [`%files`](https://github.com/os-autoinst/openQA/blob/6ca685738c5db7ccdd9394dec8d4841a293c65ec/dist/rpm/openQA.spec#L642) sections.

**Review by baierjan — CHANGES_REQUESTED** [review]

File: dist/rpm/os-autoinst-scripts-deps.spec (hunk 2 above)
Comment (perlpunk) [reply dist/rpm/os-autoinst-scripts-deps.spec:45]: ah right, the description.
I added both now. `%files` is just empty, but it's probably needed.

Comment (okurz) [reply dist/rpm/os-autoinst-scripts-deps.spec:45]: yes, an empty section is needed. Current version looks good now

**Review by okurz — APPROVED** [review]

**Review by baierjan — APPROVED** [review]

**Pull request merged**